_integrations/index.md

```diff
@@ -8,61 +8,126 @@ permalink: /integrations/
+#### Data Ingestion for Integrations
 
-Learn how to do the following using the OpenSearch Dashboards interface:
+Integrations and its dashboards depend on data ingested into OpenSearch which corresponds with the supported schema, such data pipeline that are compatible:
 
-- Access integrations
-- View integrations
-- Add integrations 
+- [OpenSearch Data-Prepper](https://github.com/opensearch-project/data-prepper)
+- [OTEL collector](https://github.com/open-telemetry/opentelemetry-collector)
+- [Fluent-Bit](https://docs.fluentbit.io/manual/pipeline/outputs/opensearch)
 
-The following image gives you a snapshot of the Integrations interface: 
+Each one of these pipelines support the OTEL schema (or directly using the simple schema ) so that they index the signal documents into the [correct index](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/schema/observability/Naming-convention.md) which represents the Observed resource signals.
 
-![Get started with Integrations demo]({{site.url}}{{site.baseurl}}/images/integrations/nginx-integration.gif)
+#### Ingestion structure
 
-### Access integrations
+Each integration contains the following metadata and assets:
 
-To access integrations, open OpenSearch Dashboards and select **Integrations** from the **Management** menu. The interface displays installed and available integrations.
+* Name & description
+* Source url & License
+* Schema spec (mapping / components mapping)
+* Sample data (for try-out experience)
+* Assets (Dashboards, Index-Patterns , Queries, Alerts )
 
-## View integrations
+---
+## Using Integrations
+Integrations can be installed directly from the default catalog which is deployed in every OpenSearch release. In addition, integrations can be manually loaded using the standard Dashboard Management console and updated to reflect the most up-to-date version or new integrations releases.
 
-To view an integration, view the dashboard associated with the integration. If the integration doesn't have an associated dashboard, select the desired integration listed under the **Installed** window. View the integration details, such as assets and fields. 
+## Installing An Integration 
 
-## Add integrations
+If you have not installed any integrations, you'll be prompted to install them from the Integrations interface. Supported integrations are listed in the **Available** window and present the default Integration release catalog.  
+![]({{site.url}}{{site.baseurl}}/images/integrations/empty-installed-integrations.png)
 
-If you have not installed any integrations, you'll be prompted to install them from the Integrations interface. Supported integrations are listed in the **Available** window. 
+To add an integration, select the desired prepackaged assets. Currently, OpenSearch Integrations has two flows: Add or Try it. The following example uses the "Try it" flow:
```

Comment:
I think we should go through a real setup example instead of the sample data flow, since "Try it" is one click. It's more writing but it will pay off to show real usage. We can maybe use the otel-demo repo?

Reply:
agree but it will to too long here - probably best to create a separate page for this purpose

Reply:
Alternatively, you can add an expandable setup example by enclosing it in the details block. This helps avoid the too long issue.
```

<details markdown="block">
  <summary>
    Demo
  </summary>
  {: .text-delta}
write the demo here
</details>

```